### ISSUE TITLE
Animation data exported even with animated object hidden

### ISSUE THREAD

**Issue by rotornut44** [AsoboStudio/glTF-Blender-IO-MSFS#267]
### Current Behavior

I have a project with a few different models in it. Only one is animated. If I hide all collections except for one of the unanimated objects that I wish to export, even with "Selected Objects" set in the export settings, false animation data gets written to the unanimated object on export.

**Edit:** Only some objects seem to be affected, some of the others in my project do not export with animations either way.

### Expected Behavior

Only data for the selected object(s) should be exported.

### Steps To Reproduce

1. Open my sample "tent_temp.blend" and ensure the hangar door collections are still hidden.
2. Export the tent with Selected Objects, Animation, Shape Keys, & Skinning checked. 
3. A gltf with animation data will be exported.
4. Export the tent with Selected Objects still checked, but uncheck Animations, Shape Keys, & Skinning.
5. A gltf without animation data will be exported (how it should export by default, without needing to uncheck the Animation boxes).

### Environment

```markdown
- OS: Windows 10
- Blender: 3.3.8
- glTF-Blender-IO-MSFS: 1.3.1
```

### Anything else?

[ESD_TENT_Temp.zip](https://github.com/AsoboStudio/glTF-Blender-IO-MSFS/files/12243244/ESD_TENT_Temp.zip)

**Comment by ronh991** [comment]
Try checking Optimize Animation Size and uncheck Always sample Animations.  It seems the Khronos code code sets animations to every node, even if there are no animations.  See if that helps.  I now get a 5K gltf with those settings.

**Comment by y-khodja** [comment]
Hello,

It has been fixed in the latest version of the exporter 1.3.2 !

Thank you for your report
